ietf/doc/views_doc.py

```diff
@@ -138,12 +140,12 @@ def interesting_doc_relations(doc):
 
     return interesting_relations_that, interesting_relations_that_doc
 
-def document_main(request, name, rev=None):
+def document_main(request, name, rev=None, document_html=False):
```

Comment:
This need discussion - If we change the document main view to allow choosing what to display to be an option, we should make this generic. It'll make a big difference when we replace document_main with a better front-end.

Reply:
What do you mean by "make this generic"?

Reply:
I think ultimately a callable, but I'm not sure it should be part of the signature of the function itself. The question is if we want to switch this based on URLs, previously expressed user preferences, or sitewide configuration.

Reply:
Sorry, I'm dense - switch between _what_?